ironfish/src/blockchain/blockchain.ts

```diff
@@ -139,6 +139,10 @@ export class Blockchain {
     this._genesis = newGenesis
   }
 
+  get databaseVersion(): number {
+    return DATABASE_VERSION
```

Comment:
So this isn't exactly the DB version, which is important to point out. We don't need this setter because you can use `this.db.getVersion()`. So just to be clear, this is the software DB version, but the `db.getVersion()` is the real DB version that it's migrated to. The difference is, if you haven't started your node and run migrations then this will report a different DB version than the actual DB version of the database.